protocol/x/prices/types/types.go

```diff
@@ -43,7 +45,15 @@ type PricesKeeper interface {
 		ctx sdk.Context,
 		linearInterpolateFunc func(v0 uint64, v1 uint64, ppm uint32) (uint64, error),
 	) error
+	GetValidMarketPriceUpdates(
```

Comment:
Where is this function defined? I don't see it defined in this PR

Reply:
It is already included in the prices keeper https://github.com/dydxprotocol/v4-chain/blob/1aaa2ccdcd6757a31444673d0eb4aa0f0b674a93/protocol/x/prices/keeper/update_price.go#L31 

I'm just exposing it here so that it will show up in the generated mock.